sandbox/src/routes/retrieveOboUsers.js

```diff
@@ -8,21 +8,21 @@ module.exports = [
         method: 'GET',
         path: '/FHIR/R4/Practitioner',
         handler: (request, h) => {
-            
+
             // check that the query exists and is correct
             const query = request.query._query;
             if (query == undefined || query != "onBehalfOf") {
                 return h.file('SandboxErrorOutcome.json').code(400);
             }
-            
+
             if (request.headers["nhsd-ers-business-function"] !== "SERVICE_PROVIDER_CLINICIAN_ADMIN")
             {
-                return h.response("This endpoint can only be accessed with the SERVICE_PROVIDER_CLINICIAN_ADMIN business function").code(403);
+                return h.response('SANDBOX_ERROR: This endpoint cannot be accessed using the e-RS Business Function provided. Allowed values: SERVICE_PROVIDER_CLINICIAN_ADMIN').code(403)
             }
 
```

Comment:
Changed the error to be the same format as all other endpoints - the tests are common and require the same format